src/components/CounterBadge/CounterBadge.st.css

```diff
@@ -0,0 +1,69 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback;
+}
+
+/*Overrides*/
```

Comment:
This title could be moved to the overrideable section or you can remove it.

// here
:vars {
    /*
        Background color
        @default primaryBgColor: color-8, secondaryBgColor: color-8 with opacity 0.1
    */
    CounterBadgeBgColor: --overridable;
    /*
    .....